llvm/test/lab2/simonyan_suren/test.ll

```diff
@@ -0,0 +1,132 @@
+; RUN: opt -load-pass-plugin=%llvmshlibdir/SimonyanInliningPass%pluginext -passes=simonyan-inlining -S %s | FileCheck %s
+
```

Comment:
Could you please also add testing for the following case:
```
void foo() {
  float a = 1.0f;
  if (a < 2.0f)
    a += 1.0f;
}

void bar1() {
  int a = 0;
  if (a < 1)
    foo();
  a++;
}
```
Expected that structure of basic blocks of both functions will be kept.